### PR TITLE
Add auto-deploy option for react-codesandboxer

### PR DESCRIPTION
**Pull request by CompuIves**
This option allows a sandbox to deploy on mount. I to use this to automatically creates an iframe embed from a component.

### CODE REVIEW

File: packages/react-codesandboxer/README.md [hunk 1]
```diff
@@ -72,11 +72,11 @@ Children also receives the `sandboxId` and the `sandboxUrl` once those exist.
```
Comment (CompuIves) [inline packages/react-codesandboxer/README.md]: Ahh, it ran prettier here. I can remove the prettifying if you prefer that!

**Comment by CompuIves** [comment]
This is the idea! https://codesandbox.io/s/215z0lwj0y

File: packages/react-codesandboxer/README.md (hunk 1 above)
Comment (Noviny) [reply packages/react-codesandboxer/README.md]: I'm not fussed about removing the prettying. All good here.

**Comment by CompuIves** [comment]
> I've been making this harder to do since I didn't want you to end up with too many requests when they weren't needed

Hmm yes, that's understandable. I think that for the client sandboxes our servers will be fine with this, so no problem!

> I think it would actually be cool if, as well as adding this prop, we exported a second component from this package which was the SandboxEmbed component. 

Hah, I did something like that! (https://github.com/CompuIves/codesandbox-client/tree/master/packages/react-embed)

It's probably better to move it to this repo, I agree!

**Comment by Noviny** [comment]
I’m going to merge this pr so this just gets in my next release tonight.

Would welcome the embed pr, but happy if you want to keep owning it.

Thanks again for the contribution ❤️

Ps. Would be neat if you run `yarn changeset` on future PRs. I’m testing out a monorepo changeset management tool.

**Pull request merged**